rustls/src/client/tls13.rs

```diff
@@ -301,55 +301,44 @@ pub fn prepare_resumption(
     exts: &mut Vec<ClientExtension>,
     doing_retry: bool,
 ) -> bool {
-    let resuming_suite = handshake
-        .resuming_session
-        .as_ref()
-        .and_then(|resume| sess.find_cipher_suite(resume.cipher_suite));
-
-    if hs::compatible_suite(sess, resuming_suite) {
-        sess.resumption_ciphersuite = resuming_suite;
-        // The EarlyData extension MUST be supplied together with the
-        // PreSharedKey extension.
-        let max_early_data_size = handshake
-            .resuming_session
-            .as_ref()
-            .map_or(0, |resume| resume.max_early_data_size);
-        if sess.config.enable_early_data && max_early_data_size > 0 && !doing_retry {
-            sess.early_data
-                .enable(max_early_data_size as usize);
-            exts.push(ClientExtension::EarlyData);
-        }
+    let resuming_session = if let Some(resuming_session) = handshake.resuming_session.as_ref() {
+        resuming_session
+    } else {
+        return false;
+    };
 
-        // Finally, and only for TLS1.3 with a ticket resumption, include a binder
-        // for our ticket.  This must go last.
-        //
-        // Include an empty binder. It gets filled in below because it depends on
-        // the message it's contained in (!!!).
-        let (obfuscated_ticket_age, suite) = {
-            let resuming = handshake
-                .resuming_session
-                .as_ref()
-                .unwrap();
-            (
-                resuming.get_obfuscated_ticket_age(ticketer::timebase()),
-                resuming.cipher_suite,
-            )
-        };
+    let resuming_suite = match sess.find_cipher_suite(resuming_session.cipher_suite) {
+        Some(resuming_suite) if hs::compatible_suite(sess, resuming_suite) => resuming_suite,
+        _ => {
+            return false;
+        }
+    };
```

Comment:
And here you already did it like that (though stable rustfmt would format as `_ => return false,`, which is preferable, IMO).

Reply:
I think we should just let whoever can rustfmt stuff do the formatting. I can't guess what rustfmt 2 would do, especially I don't know what the brace style would be.